turbopack/crates/turbopack-ecmascript/src/analyzer/graph.rs

```diff
@@ -737,6 +737,8 @@ impl EvalContext {
                 ..
             }) => JsValue::WellKnownObject(WellKnownObjectKind::ImportMeta),
 
+            Expr::Assign(AssignExpr { right, .. }) => self.eval(right),
```

Comment:
if i have `a = (b = <expr> )`  does 'expr' end up geting eval'd twice due to this?

visit_assign_expr calls `eval' on the RHS and then calls `visit_expr` on the rhs which will visit the rhs again.

this probably isn't a big deal, but i guess it points at a fundamental redundancy between this recursive function and the traversal itself.

Reply:
Yes, it would do it twice, unfortunately.